Frontend/src/app/home/filter-dialog/filter-dialog.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import {
+	MAT_DIALOG_DATA,
+	MatDialogActions,
+	MatDialogContent,
+	MatDialogRef,
+	MatDialogTitle,
+} from '@angular/material/dialog';
+import { FilterDialogData } from '../../model/filter-dialog-data.model';
+import { MatActionList, MatList, MatListItem } from '@angular/material/list';
+import { MatFormField, MatOption, MatSelect } from '@angular/material/select';
+import {
+	FormBuilder,
+	FormControl,
+	FormGroup,
+	ReactiveFormsModule,
+} from '@angular/forms';
+import { MatButton } from '@angular/material/button';
+
+@Component({
+	selector: 'app-filter-dialog',
+	standalone: true,
+	imports: [
+		MatList,
+		MatSelect,
+		MatFormField,
+		MatOption,
+		ReactiveFormsModule,
+		MatDialogContent,
+		MatDialogTitle,
+		MatActionList,
+		MatListItem,
+		MatDialogActions,
+		MatButton,
+	],
+	templateUrl: './filter-dialog.component.html',
+	styleUrl: './filter-dialog.component.scss',
+})
+export class FilterDialogComponent implements OnInit {
+	protected formGroup: FormGroup;
+	protected priceCategoryControl = new FormControl('');
```

Comment:
Is this necessary here, you will make a new FormControl in the `group` method no?